var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -492,6 +495,18 @@ def bjam_python_line(self, spec):
             spec["python"].libs[0],
         )
 
+    def bjam_python_line_win(self, spec):
+        # avoid "ambiguous key" error
+        if spec.satisfies("@:1.58"):
+            return ""
+
+        return "using python : {0} : {1} : {2} : {3} ;\n".format(
+            spec["python"].version.up_to(2),
+            spec["python"].command.path.replace("\\", "/"),
+            spec["python"].prefix.include.replace("\\", "/"),
+            spec["python"].prefix.libs.replace("\\", "/"),
```

Comment:
(minor) I think it might be slightly more robust to do `pathlib.Path(...).as_posix`